Sming/Libraries/ThingSpeak/README.md

```diff
@@ -0,0 +1,61 @@
+# Sming - ThingSpeak driver libary
```

Comment:
The text needs style and syntax corrections. May be @riban-bw can help you here?